Ruby_AB2014.md

```diff
@@ -27,9 +27,9 @@
 
 - **Ruby.philosopy[1] **
 
-Perl'den daha güçlü Phyton'dan daha fazla nesne yönelimli betik bir dildir.
+Perl'den daha güçlü Pyhton'dan daha fazla nesne yönelimli betik bir dildir.
 
-> Perl + Smalltalk + Phyton + Lisp + Ada + ... = Ruby 
+> Perl + Smalltalk + Pyhton + Lisp + Ada + ... = Ruby 

```

Comment:
Thank you for your PR! I merged before PR also the same thing inside. instead of `Pyhton` should be `Python` ;)